### PR TITLE
Introduced ApplicationContext/Boot

### PR DESCRIPTION
**Pull request by pperalta**
Servers are started with dedicated *Application
classes that create an application context and
start their respective servers via Boot.

https://github.com/markfisher/xdzk/issues/13

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,20 +1,45 @@
+buildscript {
+	ext {
+		springBootVersion = '1.0.0.BUILD-SNAPSHOT'
+	}
+	repositories {
+		mavenLocal()
```
Comment (markfisher) [inline build.gradle]: Did you intend to commit with mavenLocal here?

File: src/main/java/xdzk/server/AdminApplication.java [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+		return new MapBytesUtility();
+	}
+
+	/**
+	 * Bean definition for singleton instance of {@link ContainerMatcher}.
+	 *
+	 * @return new instance of ContainerMatcher
+	 */
+	@Bean
+	public ContainerMatcher containerMatcher() {
```
Comment (markfisher) [inline src/main/java/xdzk/server/AdminApplication.java]: I don't think is being used yet. It's not passed to the AdminServer.

File: src/main/java/xdzk/server/ContainerApplication.java [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+		return new MapBytesUtility();
+	}
+
+	/**
+	 * Bean definition for singleton instance of {@link ContainerMatcher}.
+	 *
+	 * @return new instance of ContainerMatcher
+	 */
+	@Bean
+	public ContainerMatcher containerMatcher() {
```
Comment (markfisher) [inline src/main/java/xdzk/server/ContainerApplication.java]: This is only needed for an AdminServer right?

File: build.gradle (hunk 1 above)
Comment (pperalta) [inline build.gradle]: This was a copy/paste from the `build.gradle` generated by http://start.spring.io/. I'll get rid of it.

File: src/main/java/xdzk/server/AdminApplication.java (hunk 2 above)
Comment (pperalta) [inline src/main/java/xdzk/server/AdminApplication.java]: You are correct. I'll get rid of the `@Bean`. This requires more refactoring anyway (as I mentioned the `ContainerMatcher` should be provided by the `Stream`. I'll do that refactoring in a future commit.

File: src/main/java/xdzk/server/ContainerApplication.java (hunk 3 above)
Comment (pperalta) [inline src/main/java/xdzk/server/ContainerApplication.java]: You can tell I did this in a rush. :-/  Tons of copy/paste errors!

**Pull request merged**